### ISSUE TITLE
Mermaid Pie Chart gets cut off

### ISSUE THREAD

**Issue by Daenges** [laurent22/joplin#6769]
![mermaid-1660924804126](https://user-images.githubusercontent.com/57369924/185659881-a1963e02-4503-4565-8bcc-02babef5466f.png)

The legend of Mermaid pie charts is truncated. This also applies to the editor, HTML and PNG export.
Zoom level change does not fix this for me.

## Environment

Joplin version: 2.8.8 Flatpak (Flathub)
Platform: Linux
OS specifics: Manjaro

## Steps to reproduce
Here is a sample Graph, used to produce the image.
```
pie showData 
	title Anlageportfolio
	"Anleihen": 5000
	"Aktien": 5000
```

## Describe what you expected to happen
Display a full graph.

## Logfile
[Console.log](https://github.com/laurent22/joplin/files/9383669/Console.log)

**Comment by tessus** [comment]
I could reproduce this. The Live Editor shows it [correctly](https://mermaid.live/edit#pako:eNpFj70KwzAMhF_FaM6QpYu3QtZO3YoWESuJqW0ZR6aUkHev0xYy3nc_cBuM4hgsZM9mXeQ1kJLBhKpeA5trCjRzlqKTBC-HgdAg-4UTgjWXvu__9Kn-ZNBB5BLJuza-YTIGQReOfAQQHE9UQ2th2luUqsr9nUawWip3ULMj5cHTXCiCnSisjWZKD5FTs_Mq5fY78P2xfwDtv0a5).

**Comment by Daenges** [comment]
![mermaid-1664209358048](https://user-images.githubusercontent.com/57369924/192330424-cd5cd4f3-8a12-46ad-b47f-27601973b86e.png)

Same applies to the labeling of flowchart arrows.

**Comment by Daenges** [comment]
This issue still persists.

**Comment by adarsh-sgh** [comment]
> I could reproduce this. The Live Editor shows it [correctly](https://mermaid.live/edit#pako:eNpFj70KwzAMhF_FaM6QpYu3QtZO3YoWESuJqW0ZR6aUkHev0xYy3nc_cBuM4hgsZM9mXeQ1kJLBhKpeA5trCjRzlqKTBC-HgdAg-4UTgjWXvu__9Kn-ZNBB5BLJuza-YTIGQReOfAQQHE9UQ2th2luUqsr9nUawWip3ULMj5cHTXCiCnSisjWZKD5FTs_Mq5fY78P2xfwDtv0a5).

Here is another test on live editor renderer opened in full window
1. open [live editor rendered window](https://mermaid.live/view#pako:eNpFjj0PgzAMRP9K5JmBpQtbJdZO3SovFjFgNYmjYFRViP_e0A91fffudBsM6hk6yMJumfXRk5HDhGZigd05BZo4a7FRg-gRIFTIMnNC6NypbdsvvZv8GTQQuUQSX8c3TM4h2MyRDwHB80hrqC1Me1VpNb0-0wCdlZUbWLMn415oKhR_kL2Ylsvn7_t2A5nSTbUqI4WF9xeipUZu)

2. resize the window to a lower the width (to 500px-600px )
3. refresh the page 

re-rendered graph gets cut off the same way as it gets cut in our app, looks like a problem from mermaid's side; 
viewbox width is being set lower than requiered in this case.

**Comment by tessus** [comment]
Maybe you can open a ticket in the mermaid repo?

**Comment by adarsh-sgh** [comment]
> Maybe you can open a ticket in the mermaid repo?

Sure, will do it.

edit: https://github.com/mermaid-js/mermaid/issues/4082

**Comment by tessus** [comment]
Fixed by #8890
